modules/migration-downloading-logs.adoc

```diff
@@ -11,14 +11,16 @@ You can download the `Velero`, `Restic`, and `MigrationController` pod logs in t
 
 .Procedure
 
-. Log in to the {mtc-short} console.
-. Click *Plans* to view the list of migration plans.
-. Click the *Options* menu {kebab} of a specific migration plan and select *Logs*.
-. Click *Download Logs* to download the `MigrationController`, `Velero`, and `Restic` pod logs for all clusters.
-. To download a single log, specify the following options:
+. In the {mtc-short} console, click *Migration plans* to view the list of migration plans.
+. Click the Options menu {kebab} of a specific migration plan and select *Logs*.
+. Click *Download Logs* to download the logs of the `MigrationController`, `Velero`, and `Restic` pods for all clusters.
```

Comment:
Consider making the rest of the steps options of a single step. It's unlikely that a user's going to do all of them, and the way they are now, it looks like you need to do all of them in sequence.

Reply:
I'm not sure how this is done in OpenShift docs. I updated this section. Let me know if this works.